concordium-sdk/src/main/java/com/concordium/sdk/responses/transactionstatus/ContractInitializedResult.java

```diff
@@ -65,11 +71,12 @@ public static ContractInitializedResult from(ContractInitializedEvent contractIn
                 .amount(CCDAmount.from(contractInitialized.getAmount()))
                 .version(ContractVersion.from(contractInitialized.getContractVersion()))
                 .initName(contractInitialized.getInitName().getValue())
+                .parameter(Parameter.from(contractInitialized.getParameter()))
```

Comment:
This has nothing to do with p8 I assume?

Reply:
Yep, this is a part of GRPC changelog, back then I assumed everything new in GRPC is related to Protocol 8